src/views/Viewer.vue

```diff
@@ -674,6 +695,18 @@ export default {
 				this.sidebarWidth = sidebar.offsetWidth
 			}
 		},
+
+		onPrev(info, oldFileInfo) {
+			this.Viewer.onPrev(info, oldFileInfo)
+		},
+
+		onNext(info, oldFileInfo) {
+			this.Viewer.onNext(oldFileInfo)
```

Comment:
Hey @azul, I just rezlized we did a mistake, the onPrev and onNext have different parametters.
It should most likely be consistent:
```js
this.Viewer.onNext(oldFileInfo)
```